cuebot/src/main/java/com/imageworks/spcue/dao/postgres/DispatchQuery.java

```diff
@@ -23,11 +23,23 @@
 
     public static final String FIND_JOBS_BY_SHOW =
         "/* FIND_JOBS_BY_SHOW */ " +
-        "SELECT pk_job, int_priority, rank FROM ( " +
+            "SELECT pk_job, int_priority, rank FROM ( " +
             "SELECT " +
-                "ROW_NUMBER() OVER (ORDER BY job_resource.int_priority DESC) AS rank, " +
-                "job.pk_job, " +
-                "job_resource.int_priority " +
+                "ROW_NUMBER() OVER (ORDER BY int_priority DESC) AS rank, " +
+                "pk_job, " +
+                "int_priority " +
+            "FROM ( " +
+            "SELECT " +
+                "job.pk_job as pk_job, " +
+                "/* sort = priority + (100 * (1 - (job.cores/job.int_min_cores))) + (age in days) */ " +
```

Comment:
Can this line be removed? Or maybe, if it's meant to explain the following code, make it a bit more clear that it's a comment.